modules/user_accounts/php/edit_user.class.inc

```diff
@@ -304,14 +305,30 @@ class Edit_User extends \NDB_Form
                      'userID'      => $uid,
                     )
                 );
-                $examinerID = $examinerID = $DB->pselectOne(
+                $examinerID = $DB->pselectOne(
                     "SELECT examinerID
-                 FROM examiners
-                 WHERE full_name=:fullName",
+                            FROM examiners
+                            WHERE full_name=:fullName",
                     array('fullName' => $values['Real_name'])
                 );
-            } else {
-
+//            } elseif (!empty($examinerID)
+//                && ((!empty($ex_radiologist)
+//                && !empty($ex_pending)
+//                && !empty($ex_curr_sites))
+//                || (empty($ex_radiologist)
+//                && empty($ex_pending)
+//                && empty($ex_curr_sites)))
+//            ) {
+                } elseif (!empty($examinerID)
+                  && ((!empty($ex_radiologist)
+                  && !empty($ex_pending)
+                  && !empty($ex_curr_sites))
+                  || empty($ex_curr_sites))
```

Comment:
@cmadjar, addressing your most recent change, shouldn't the else if statement be:
```
else if (!empty($examinerID)
		 && !empty($ex_pending) 
		 && !empty($ex_radiologist)) 
		 && (empty($ex_curr_sites) || !empty($ex_curr_sites))
		) {

		 	// The case where the examiner already exists 
		 	// and has either all sites being deleted or the sites are being changed
		  }
```

Reply:
Hi @um4r12 ,

Thanks for the sugesstion. Actually, I tried that and it does not work as it is impossible to be in the following condition `!empty($examinerID) && !empty($ex_pending) && !empty($ex_radiologist)) && empty($ex_curr_sites)` in the frontend. To remove all the sites from an examiner, you have to unselect radiologist and ex_pending. Currently, only lets you be in either of the following conditions:
- `!empty($examinerID) && !empty($ex_pending) && !empty($ex_radiologist)) && !empty($ex_curr_sites)` 
- `!empty($examinerID) && empty($ex_pending) && empty($ex_radiologist)) && empty($ex_curr_sites)` which I simplified in my last commit by `!empty($examinerID) && empty($ex_curr_sites)`

Reply:
@cmadjar is correct, its not ideal but is acceptable in the scope of this bugfix. a revap of the logic is def needed to prevent unsetting of all fields and to sync the validation checks to the processing.